### ISSUE TITLE
Specify electron version

### ISSUE THREAD

**Issue by jcvernaleo** [decred/decrediton#345]
Our current build method used node v6.9.5 (which is  NODE_MODULE_VERSION 50).  A fresh npm install grabs electron 1.6.5 (NODE_MODULE_VERSION 53).  For some reason npm thinks those are compatible with each other (^ when specifying versions).  Need to force a version of electron so builds continue to work.

**Comment by peterzen** [comment]
I got a clean build on 1.6.5 and node v7.9.0 after building `grpc` from source (the included binary is incompatible with node < 53 and seems to unleash a bit of a dependency hell).  [This script](https://github.com/peterzen/decrediton/blob/docker-ci-builder/build-docker.sh) has an example of how to make it work.  (Be sure to `rm -rf ~/.electron*` after changing your `node` version.)

**Comment by jcvernaleo** [comment]
I had no doubt that a newer node would work but this close to a release I did not want to go and upgrade node versions and electron versions.  Plus I think forcing a specific version of electron (as I did in #346) is a better way to go since I've seen way too many times where minor version changes break everything with node.  I very much prefer doing vendoring the way we do in go with glide than the npm version guessing.
